content/sensu-go/6.5/observability-pipeline/observe-schedule/backend.md

```diff
@@ -120,29 +121,91 @@ If you did not use environment variables to override the default admin credentia
 
 ### Ubuntu/Debian or RHEL/CentOS initialization
 
-For Ubuntu/Debian or RHEL/CentOS, set administrator credentials with environment variables at [initialization][25] as shown below, replacing `YOUR_USERNAME` and `YOUR_PASSWORD` with the username and password you want to use:
+For Ubuntu/Debian or RHEL/CentOS, set administrator credentials with environment variables at [initialization][25] as shown below.
+
+To initialize with your username and password, replace `<username>` and `<password>` with the username and password you want to use:
 
 {{< code shell >}}
-export SENSU_BACKEND_CLUSTER_ADMIN_USERNAME=YOUR_USERNAME
-export SENSU_BACKEND_CLUSTER_ADMIN_PASSWORD=YOUR_PASSWORD
+export SENSU_BACKEND_CLUSTER_ADMIN_USERNAME=<username>
+export SENSU_BACKEND_CLUSTER_ADMIN_PASSWORD=<password>
 sensu-backend init
 {{< /code >}}
 
 {{% notice note %}}
 **NOTE**: Make sure the Sensu backend is running before you run `sensu-backend init`.
 {{% /notice %}}
 
+### Add API key for initialization
+
+Add an API key when you initialize the backend to make automated cluster setup and deployment more straightforward.
+For example, if you supply an API key via sensu-backend init, you do not need to configure sensuctl.
+Instead, you can execute sensuctl commands to manage resources immediately after initializing a cluster &mdash; just provide the [`--api-key` and `--api-url` flags][62] with their correct values in your sensuctl commands.
```

Comment:
It should be explained that the API key must be a valid UUID as per https://en.wikipedia.org/wiki/Universally_unique_identifier